dockstore-integration-testing/src/test/java/io/dockstore/webservice/WebhookIT.java

```diff
@@ -278,4 +285,83 @@ private Long getNullVisibilityCount() {
         return testingPostgres.runSelectStatement(
             "select count(*) from workflow where gitvisibility is null", long.class);
     }
+
+    @Test
+    void testEntryNamesInLambdaEvents() {
+        final ApiClient webClient = getOpenAPIWebClient(BasicIT.USER_2_USERNAME, testingPostgres);
+        final WorkflowsApi workflowsApi = new WorkflowsApi(webClient);
+        final LambdaEventsApi lambdaEventsApi = new LambdaEventsApi(webClient);
+        // Constants for the "dockstore-testing/workflow-dockstore-yml" repo
+        final String tag01 = "refs/tags/0.1";
+        final String tag02 = "refs/tags/0.2";
+        final String branchInvalidDockstoreYml = "refs/heads/invalidDockstoreYml";
+        final String branchDifferentLanguagesWithSameWorkflowName = "refs/heads/differentLanguagesWithSameWorkflowName";
+        final String foobarWorkflowName = "foobar";
+        final String foobar2WorkflowName = "foobar2";
+
+        // Track install event
+        workflowsApi.handleGitHubInstallation(installationId, workflowDockstoreYmlRepo, BasicIT.USER_2_USERNAME);
```

Comment:
This GitHub installation was throwing an exception because it attempts to perform a GitHub release on a branch that contains an invalid .dockstore.yml